lib/asciidoctor.rb

```diff
@@ -501,7 +500,7 @@ class << self; self; end.send :attr_accessor, key
     #                collapsing the line breaks and indentation to
     #                a single space.
     #
-    AttributeEntryRx = /^:(!?\w.*?):(?:#{CG_BLANK}+(.*))?$/
+    AttributeEntryRx = /^:(!?\w.*?):(?:\s+(.*))?$/
```

Comment:
> /\s/ - A whitespace character: /[ \t\r\n\f]/

So if I'm reading this regular expression correctly, attribute value can be defined on a new line ?

```
:foo:
bar
```